.github/workflows/pr.yml

```diff
@@ -14,15 +14,7 @@ jobs:
       matrix:
         java: [ '11' ]
         scala: [
-            { version: '2.12.13' },
-            { version: '2.12.12' },
-            { version: '2.12.11' },
-            { version: '2.12.10' },
-            { version: '2.13.4' },
-            { version: '2.13.3' },
-            { version: '2.13.2' },
-            { version: '2.13.1' },
-            { version: '2.13.0' }
+            { version: '2.12.13' }
```

Comment:
We need to change this all up in the future. We can't run a matrix like this since it's not doing what we want. Instead in the actual scripted tests we'll want to instead run `+test` in the test file. This is trying to run the actual plugin in all these versions, which won't work.